main.go

```diff
@@ -61,17 +74,67 @@ func main() {
 	dg.Close()
 }
 
+func paramsTemplate() url.Values {
+	params := url.Values{}
+	params.Set("action", "query")
+	params.Set("format", "json")
+	return params
+}
+
+func getWikipediaAPI(params url.Values) (*http.Response, error) {
+	// APIにリクエストを送信
+
+	resp, err := http.Get(endpoint + "?" + params.Encode())
+	if err != nil {
+		return resp, err
+	}
+
+	return resp, err
+}
+
+func getDiscordMessage() (string, error) {
+	params := paramsTemplate()
+	params.Set("list", "search")
+	pageTitle, pageURL, err := getRandomPage()
+	if err != nil {
+		log.Fatal("Error getting Wikipedia page: ", err)
+		return "", err
+	}
+	params.Set("srsearch", pageTitle)
+	params.Set("srlimit", "1")
+
+	resp, err := getWikipediaAPI(params)
+	defer resp.Body.Close()
+
+	var result WikipediaSearchResult
+	if err := json.NewDecoder(resp.Body).Decode(&result); err != nil {
+		return "", err
+	}
+	var content string
+	if len(result.Query.Search) == 0 {
+		content = pageTitle + "\n" + "<" + pageURL + ">"
```

Comment:
```suggestion
	if len(result.Query.Search) == 0 {
		content := pageTitle + "\n" + "<" + pageURL + ">"
```
同上で、ここの`var`宣言は冗長なので`:=`に置き換えてほしいです。